Source/ASDisplayNode.mm

```diff
@@ -438,36 +438,39 @@ - (void)onDidLoad:(ASDisplayNodeDidLoadBlock)body
 - (void)asyncTraitCollectionDidChangeWithPreviousTraitCollection:(ASPrimitiveTraitCollection)previousTraitCollection
 {
   if (@available(iOS 13.0, *)) {
-    __instanceLock__.lock();
+    // When changing between light and dark mode, often the entire node needs to re-render.
+    // This change doesn't happen frequently so it's fairly safe to render nodes again
     if (self.primitiveTraitCollection.userInterfaceStyle != previousTraitCollection.userInterfaceStyle) {
-      // When changing between light and dark mode, often the entire node needs to re-render.
-      // This change doesn't happen frequently so it's fairly safe to render nodes again
-      BOOL needsClippingCornerUpdate = NO;
-      CGFloat cornerRadius = _cornerRadius;
-      ASCornerRoundingType cornerRoundingType = _cornerRoundingType;
-      UIColor *backgroundColor = _backgroundColor;
       if (_loaded(self)) {
```

Comment:
`-loaded()` needs to be called with the lock held. You can call it while holding the lock to get `_primitiveTraitCollection` (instead of calling the getter).